pkg/configuration/meta_parameters.go

```diff
@@ -0,0 +1,24 @@
+// See parameters.go for this package's comment
+package configuration
+
+// Meta parameters are parameters not directly specified by the user's input. They indicate the completeness of
+// the input parameters and the readiness of the application to begin its execution.
+type MetaParameters struct {
+	parsed           bool // Indicates whether the flag.Parse() method was called or not
+	mandatoryPresent bool // Indicates whether the mandatory parameters are persent or not
+	valid            bool // Indicates if the values received for each flags are valid or not
+
+	MultipleIssue bool // Indicates whether the resource received multiple Jira issue to process
+}
+
+// Returns true if the flags were parsed by the Go flags api (meta.parsed) and if all the mandatory parameters
+// have been assigned a value (meta.mandatoryPresent).
+func (meta *MetaParameters) AllMandatoryValuesPresent() bool {
+	return meta.parsed && meta.mandatoryPresent
+}
+
+// Returns true if the flags were parsed by the Go flags api (meta.parsed) and if the set of parameters received
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [comment on exported method MetaParameters.Ready should be of the form "Ready ..."](https://app.codacy.com/app/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4570750)